config.yaml

```diff
@@ -17,15 +17,14 @@ jobs:
           type: "NUMBER"
     destination:
       ref: postgres
-      table_name: "parameterized_results_4238114"
+      table_name: "results_4238114"
       if_exists: "replace"
 
-  - name: Some other job
+  - name: Write Results of Job 1 back
     source:
       ref: postgres
-      table_name: foo_table
       if_exists: "append"
-      query_string: "SELECT * FROM results_4194034;"
+      query_string: "SELECT * FROM results_4238114;"
```

Comment:
The purpose of this sample file is that it actually runs. the second job must read from the first job so the table exists!